aiida/cmdline/commands/node.py

```diff
@@ -351,8 +351,12 @@ def run(self, *args):
         for key, req_vals in (('id', parsed_args.pks), ('uuid', parsed_args.uuids)):
             if req_vals:
                 filters['or'].append({key:{'in':req_vals}})
-
-
+        
+        # If neither of the arguments is passed, exit the program
+        if not filters['or']:
+            print "Nothing to delete"
```

Comment:
Please change this to
```suggestion
            print "No nodes specified."
```